lib/mail_room.rb

```diff
@@ -2,16 +2,7 @@
 require 'optparse'
 require 'yaml'
 
-# The MailRoom namespace
 module MailRoom
-  # The MailRoom Delivery namespace holds any defined delivery methods
-  #   including:
-  # 
-  #   * postback (default)
-  #   * letter_opener
-  #   * logger
-  #   * noop
-  module Delivery; end
```

Comment:
I moved the definition of `module Delivery` to `lib/delivery.rb`, so it wasn't needed here anymore.